release/restcomm-assembly-as7/pom.xml

```diff
@@ -0,0 +1,257 @@
+                                            <artifactId>server</artifactId>
+                                            <type>zip</type>
+                                            <classifier>assembly</classifier>
+                                        </artifactItem>
+                                    </artifactItems>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-assembly-plugin</artifactId>
+                        <version>3.1.0</version>
+                        <configuration>
+                            <descriptors>
+                                <descriptor>src/main/resources/descriptor_all-in-one.xml</descriptor>
+                            </descriptors>
+                            <finalName>Restcomm-JBoss-AS7-${project.version}</finalName>
+                            <appendAssemblyId>false</appendAssemblyId>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>single</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>0.25.2</version>
+
+                        <configuration>
+                            <images>
+                                <image>
+                                    <name>restcomm/restcomm</name>
+                                    <build>
+                                        <tags>
+                                            <tag>${project.version}</tag>
+                                            <tag>latest</tag>
+                                        </tags>
+                                        <filter>${*}</filter>
+                                        <dockerFile>../../../Dockerfile_all-in-one</dockerFile>
+                                    </build>
+                                </image>
+                            </images>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>start</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <!-- "build" should be used to create the images with the
+                                         artifact -->
+                                    <goal>build</goal>
```

Comment:
again i dont see the "build" goal being hooked to pre-integration-test. i think "package" would be better

Reply:
@jaimecasero that is a fair point. 

btw, do you agree that it _should_ be bound to some phase, or would you prefer that we added the plugin and then explicitly invoke the `docker:build` maven target when we want to build a docker image?

Reply:
im looking into hook docker image generation into some phase, so its part of "regular" build, and invoked with regular maven goals